companion/src/modeledit/modeledit.cpp

```diff
@@ -120,9 +120,14 @@ void ModelEdit::on_pushButton_clicked()
 
 void ModelEdit::launchSimulation()
 {
-  RadioData *simuData = new RadioData();
+  RadioData * simuData = new RadioData();
   simuData->generalSettings = generalSettings;
-  simuData->models[0] = model;
+  if (simuData->models.size() == 0) {
+    simuData->models.push_back(model);
+  }
+  else {
+    simuData->models[0] = model;
+  }
```

Comment:
When launched this way, the simulator will only see one model - the one being edited. All other models are not transfered into simuData. Is this intentional? Perhaps it was like this before the storage changes, but I think not.